environment/src/test/java/com/sequenceiq/environment/environment/flow/deletion/handler/ResourceEncryptionDeleteHandlerTest.java

```diff
@@ -68,6 +79,15 @@ void setUp() {
                 .withId(ENVIRONMENT_ID)
                 .withResourceCrn(ENVIRONMENT_CRN)
                 .withName(ENVIRONMENT_NAME)
+                .withCloudPlatform("AZURE")
+                .withParameters(ParametersDto.builder()
+                        .withAzureParameters(AzureParametersDto.builder()
+                                .withEncryptionParameters(AzureResourceEncryptionParametersDto.builder()
+                                        .withDiskEncryptionSetId("/subscriptions/dummySubscriptionId/resourceGroups/dummyResourceGroup/" +
+                                                "providers/Microsoft.Compute/diskEncryptionSets/dummyDesId")
+                                        .build())
+                                .build())
+                        .build())
                 .build();
         EnvironmentDeletionDto build = EnvironmentDeletionDto
```

Comment:
`build` is not an expressive name; something else like `eventData` would look better.